lib/main_containers.pm

```diff
@@ -0,0 +1,78 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: module loader of container tests in SLES
+# Maintainer: qa-c@suse.de
+
+package main_containers;
+use base 'Exporter';
+use Exporter;
+use utils;
+use version_utils;
+use main_common qw(loadtest boot_hdd_image);
+use testapi qw(check_var get_required_var get_var);
+use Utils::Architectures;
+use strict;
+use warnings;
+
+
+our @EXPORT = qw(
+  is_container_test
+  load_container_tests
+);
+
+sub is_container_test {
+    return (check_var('EXTRATEST', 'cont'));
+}
+
+sub load_container_tests {
+    my $runtime = get_required_var('CONTAINER_RUNTIME');
+    boot_hdd_image;
+
+    # Container Image tests
+    if (get_var('FLAVOR', '') =~ /Container-Image/) {
```

Comment:
just a flavor... 
I would go with subroutines from day 0 here,  like:

```perl
boot_hdd_image;

if (get_var('FLAVOR', '') =~ /Container-Image/) {
    if ($runtime eq 'podman') {
        load_tests_podman_container_image();
    } else {
        load_tests_docker_container_image();
    }
} else {
    if ($runtime eq 'podman') {
        load_tests_podman_common();
    } else {
        load_tests_docker_common();
    }
}
loadtest 'console/coredump_collect';
```

Reply:
I like that. Will do it this way. Thanks!

Reply:
done